website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -364,6 +364,7 @@ This endpoint supports pagination [with standard URL query parameters](/terrafor
 | `filter[agent_pool_names]` | A comma-separated list of agent pool names. The result lists runs that use one of the agent pools you specify. | Optional |                                                                      
 | `filter[source]`           | A comma-separated list of run sources. The result lists runs that came from one of the sources you specify. Options are listed in [Run Sources](/terraform/enterprise/api-docs/run#run-sources).                               | Optional |
 | `filter[status_group]`     | A single status group. The result lists runs whose status falls under this status group. For details on options, refer to [Run status groups](/terraform/enterprise/api-docs/run#run-status-groups).                            | Optional |
+| `filter[timeframe]`        | A single integer year period. The result lists runs that were created within the year you specify. An integer year or the string "year" for the past year are valid values. If ommited, the endpoint returns all runs since the creation of the workspace.      | Optional |
```

Comment:
```suggestion
| `filter[timeframe]`        | A single integer year period. The result lists runs that were created within the year you specify. An integer year or the string "year" for the past year are valid values. If omitted, the endpoint returns all runs since the creation of the workspace.      | Optional |
```